pvanalytics/tests/features/test_daytime.py

```diff
@@ -20,6 +25,67 @@ def clearsky_january(request, albuquerque):
     )
 
 
+@pytest.fixture
+def ac_power_series():
+    # Pull down the saved PVLib dataframe and process it
+    time_series = pd.read_csv(test_file_1,
+                              parse_dates=True,
+                              index_col=0).squeeze()
+    return time_series
+
+
+@pytest.fixture
+def modeled_midday_series(ac_power_series):
+    # Get the modeled sunrise and sunset for the location
+    modeled_sunrise_sunset_df = pvlib.solarposition.sun_rise_set_transit_spa(
+        ac_power_series.index, 39.742, -105.1727)
+    modeled_sunrise_sunset_df.index = modeled_sunrise_sunset_df.index.date
+    modeled_sunrise_sunset_df = modeled_sunrise_sunset_df.drop_duplicates()
+    # Calculate the midday point between sunrise and sunset for each day
+    # in the modeled irradiance series
+    modeled_midday_series = modeled_sunrise_sunset_df['sunrise'] + \
+        (modeled_sunrise_sunset_df['sunset'] -
+         modeled_sunrise_sunset_df['sunrise']) / 2
```

Comment:
FYI this midday calculation is probably pretty close (under 60s difference, I'd expect) to the `transit` column returned from the SPA function.

Reply:
Converted to transit column in https://github.com/pvlib/pvanalytics/pull/187/commits/a74e805c6ebc275456eb4ca23758eefce5f394f2

Reply:
Updated in https://github.com/pvlib/pvanalytics/pull/187/commits/a74e805c6ebc275456eb4ca23758eefce5f394f2

Reply:
Updated in https://github.com/pvlib/pvanalytics/pull/187/commits/a74e805c6ebc275456eb4ca23758eefce5f394f2